### PR TITLE
[PassKit] Update bindings to Xcode 11.4 Beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**
Diff: https://github.com/xamarin/xamarin-macios/wiki/PassKit-iOS-xcode11.4-beta3

### CODE REVIEW

File: src/PassKit/PKEnums.cs [hunk 1]
```diff
@@ -44,8 +44,10 @@ public enum PKPassLibraryAddPassesStatus : long {
 
 	[Native]
 	public enum PKPassType : ulong {
-		Barcode, Payment,
-		// Any = ~0
+		Barcode,
+		SecureElement,
+		Payment = SecureElement,
```
Comment (spouliot) [inline src/PassKit/PKEnums.cs]: should be marked as deprecated - just for symmetry with other deprecation attributes

File: src/PassKit/PKEnums.cs [hunk 2]
```diff
@@ -74,11 +76,24 @@ public enum PKPaymentAuthorizationStatus : long {
 		PinLockout
 	}
 
+	[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
```
Comment (spouliot) [inline src/PassKit/PKEnums.cs]: ^ afaik PassKit is not available on macOS

File: src/PassKit/PKEnums.cs [hunk 3]
```diff
@@ -74,11 +76,24 @@ public enum PKPaymentAuthorizationStatus : long {
...
 	[Native]
 	public enum PKPaymentPassActivationState : ulong {
 		Activated, RequiresActivation, Activating, Suspended, Deactivated
 	}
 
+	[Watch (6,2), Mac (10,15,4), iOS (13,4)]
```
Comment (spouliot) [inline src/PassKit/PKEnums.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 4]
```diff
@@ -733,10 +768,17 @@ interface PKPass : NSSecureCoding, NSCopying {
 		[Export ("passType")]
 		PKPassType PassType { get; }
 
+		[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'SecureElementPass' instead.")]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 5]
```diff
@@ -733,10 +768,17 @@ interface PKPass : NSSecureCoding, NSCopying {
...
+		[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'SecureElementPass' instead.")]
+		[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'SecureElementPass' instead.")]
 		[iOS (8,0)]
 		[Export ("paymentPass")]
 		PKPaymentPass PaymentPass { get; }
 
+		[Watch (6,2), Mac (10,15,4), iOS (13,4)]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 6]
```diff
@@ -760,30 +802,28 @@ interface PKPaymentMethod : NSSecureCoding
 		[Export ("type")]
 		PKPaymentMethodType Type { get; }
 
+		[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'SecureElementPass' instead.")]
+		[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'SecureElementPass' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'SecureElementPass' instead.")]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 7]
```diff
@@ -760,30 +802,28 @@ interface PKPaymentMethod : NSSecureCoding
...
 		[NullAllowed, Export ("paymentPass", ArgumentSemantic.Copy)]
 		PKPaymentPass PaymentPass { get; }
 
+		[Watch (6,2), Mac (10,15,4), iOS (13,4)]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 8]
```diff
@@ -760,30 +802,28 @@ interface PKPaymentMethod : NSSecureCoding
...
-
-		[Export ("deviceAccountIdentifier")]
-		string DeviceAccountIdentifier { get; }
-
-		[Export ("deviceAccountNumberSuffix")]
-		string DeviceAccountNumberSuffix { get; }
-
+		[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'PKSecureElementPass.PassActivationState' instead.")]
+		[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'PKSecureElementPass.PassActivationState' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'PKSecureElementPass.PassActivationState' instead.")]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/passkit.cs [hunk 9]
```diff
@@ -1269,4 +1309,31 @@ interface PKDisbursementVoucher {
 		[Export ("redemptionURL", ArgumentSemantic.Copy)]
 		NSUrl RedemptionUrl { get; }
 	}
+
+	[Watch (6,2), Mac (10,15,4), iOS (13,4)]
```
Comment (spouliot) [inline src/passkit.cs]: ^ afaik PassKit is not available on macOS

File: src/PassKit/PKEnums.cs [hunk 10]
```diff
@@ -74,11 +78,24 @@ public enum PKPaymentAuthorizationStatus : long {
 		PinLockout
 	}
 
+	[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'PKSecureElementPassActivationState' instead.")]
+	[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
```
Comment (dalexsoto) [inline src/PassKit/PKEnums.cs]:
```suggestion
	[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'PKSecureElementPassActivationState' instead.")]
	[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'PKSecureElementPassActivationState' instead.")]
```

File: src/PassKit/PKEnums.cs (hunk 3 above)
Comment (dalexsoto) [reply src/PassKit/PKEnums.cs]:
```suggestion
	[Watch (6,2), iOS (13,4)]
```

**Comment by dalexsoto** [comment]
@spouliot done, thank you!

**Comment by dalexsoto** [comment]
ahhh 6.4 🙄

**Pull request merged**